plugin-api/src/main/java/org/hyperledger/besu/plugin/data/Deposit.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
```

Comment:
New files should be `Copyright contributors to Hyperledger Besu`

https://wiki.hyperledger.org/display/TSC/Copyright+and+License+Policy